### DISCUSSION TITLE
xDrip via Fossil watch will not read G6

### DISCUSSION THREAD

**Discussion by robertrub** [Navid200/xDrip#45] [Watches]
![Screenshot_20210621-211222_xDrip+](https://user-images.githubusercontent.com/76626474/122815232-781d8680-d2d5-11eb-862e-e8441d24a677.jpg)
 Hi all

I have the latest nightly xDrip on a Samsung S9 (Android 10) and Fossil Q Explorer HR 1222 watch.

The connection and data transfert between phone and watch are working perfectly. But, if I set the watch as the collector, it never connects to (or reads data from) the G6 transmitter (the firefly version).

The phone reads the BGs from G6 and sends them to the watch but never the other way around.

In the logs, I see that at some time, the watch is seen as a BT headset ???. There is also the deadlock problem...

Any help or ideas to solve the problem ? TIA.

Robert

**Comment by Navid200** [comment]
I have no experience with any watches.
This is from a post on facebook about Fossil Gen 5:

First install xDrip+ on your watch searching the apps installed on your phone. Then enable wear, only wear. Change the watch quadrant to xDrip+. Wait for the full graph to display. Then enable collector and force collector ans give it 10 minutes before it starts picking up.

**Reply by Navid200** [reply]
Another:
![Screenshot 2021-06-21 230155](https://user-images.githubusercontent.com/51497406/122856398-b9696280-d2e4-11eb-98d7-a5dcc1791bf7.png)

**Reply by robertrub** [reply]
Hi

Tnx a lot for the data transfert from FB. I'm anti FB and "social" (!!) media, I wouldn't find it.

I agree on the comments on Fossil too, too much battery drain. But is still is a nice watch and does what I want ;)

It seems that it is what I did and still no go. I have all the check marks in all the boxes but no connection. Mine is gen 4 but I don't think that it matters. I think that I waited up to 17 min before giving up.

I'm in an old stone house and have BT connection problems with my Orangelink and Emalink (probably the 2.4G WiFi bouncing around ;)  ). Maybe that is the culprit. 

I'll try some more. If I get it to work, I'll post here. 

BTW, does he mention what type of phone he has ? Seems like Samsung phones need a "workaround" in xDrip... and I have an S9.

Tnx again.

Rob.

**Reply by robertrub** [reply]
Hi

Any idea what these logs from the Fossil watch say ? 

![Screenshot_20210624-163647_xDrip+](https://user-images.githubusercontent.com/76626474/123282195-91f3df00-d50a-11eb-98bb-f29ccca82dc6.jpg)

Regards

**Reply by Navid200** [reply]
I don't have experience with watches myself.
The log is related to a calibration having been empty.

But, the time sequence in the logs is backwards for the two at the top.  Is the phone time synced with the network?  How about the watch?

**Comment by psonnera** [comment]
Which sensor is that?

**Comment by robertrub** [comment]
@Navid200 : yes the phone and watch has the same time and (should) sync with the GSM service.

The "not calibrated" is nugging me since I switched from Dexcom modified app to xDrip during an active sensor. All is working ok with the BGs but I still have the menu option "initial calibration"... I wonder if this "not calibrated" status could result in my watch not reading directly from the G6 transmitter. It doesn't sound logical but as nothing else worked, the least likely option may be the valid one. I'll start a new sensor in a couple of days (I'm on day 15) and see if it changes anything.

@psonnera It is the G6 sensor with a firefly transmitter (8P...) Phone is Samsung S9 and watch Fossil explorer gen 4. It worked as a collector with MiaoMiao 1 but not with the G6.

**Reply by Navid200** [reply]
This gives you info about calibrating a Firefly G6:
https://github.com/Navid200/xDrip/wiki/Calibrate-a-G6

With respect to the watch, I am glad that @psonnera responded as he can help and I cannot.

**Reply by psonnera** [reply]
@robertrub I've seen reports of some Fossil (Gen 5) able to collect from the G6 but recently no. Usually Wear watches don't have this capacity except for the SWR50 and some patched watches. https://github.com/NightscoutFoundation/xDrip/wiki/Patching-Android-Wear-devices-for-use-with-the-G5 I know these watches work great with Libre bridges but Dexcom not really... The best solution I know is a BlueJay.

**Comment by robertrub** [comment]
Thanks a lot for the BlueJay info @psonnera 😃 

I had read about it some time ago and had forgotten about the BlueJay 😿  I have ordered one. I hope it arrives rapidly. 

Anybody want a beautiful Fossil explorer watch ? 🤣  The Fossil is much more beautiful that BlueJay (I like good looking watches) but it being not able to connect to G6 obliges me to keep my phone on me all the time 😞 

Merci bcp 👍

**Reply by Navid200** [reply]
Which one did you order?  Do you know that there is a new model?
https://www.youtube.com/watch?v=JM5cw-xVAZk

**Reply by robertrub** [reply]
The latest on their site, the GTS version.

**Reply by Navid200** [reply]
Great.
Not that I have any personal experience.  But, it's just that that is the latest and has a lot more capabilities.

**Reply by robertrub** [reply]
I'll report here when I'll receive and use it a bit.
The video talks about a companion app but couldn't find what it is. I see the BlueJay options in xDrip but no info on how to keep the watch time updated and other phone to watch reactions.

**Reply by Navid200** [reply]
You don't need a phone.  You did with the previous version in order to set it up even though it could act as a stand-alone collector.
The new one doesn't need the phone.  You can set it up right on the watch itself.

**Reply by robertrub** [reply]
Yes but I **need** a phone as I'm using AAPS 😉

**Reply by Navid200** [reply]
You can search in this room to see if anyone has already asked the same question, or any other question you may have about BlueJay:
https://gitter.im/jamorham/BlueJay
